src/Qir/Tests/QIR-static/qir-test-noqsharp.ll

```diff
@@ -9,7 +9,7 @@ declare i8* @__quantum__rt__array_get_element_ptr(%Array*, ...)
 declare i32 @__quantum__rt__array_get_dim(%Array*)
 declare %Array* @__quantum__rt__array_project(%Array*, i32, i64)
 declare void @__quantum__rt__array_update_reference_count(%Array*, i32)
-declare void @DebugLog(i64)
+;declare void @DebugLog(i64)
```

Comment:
nit: can be removed now

Reply:
Hm... I thought I removed that... Removed again ;-)